### PR TITLE
Comment with file

### PR DESCRIPTION
**Pull request by deciocandido**
**BE Comment with images**

- Adds post request to create comment with media

**User Story**
As user of Cuddly, I want to create a comment with images, so that I can comment images

As user of Cuddly, I want to create a comment with videos, so that I can share content about my pet

Screenshot 2021-05-14 at 14 26 23

### CODE REVIEW

File: src/main/java/se/kth/sda/skeleton/comments/Comment.java [hunk 1]
```diff
@@ -46,6 +46,11 @@
 
     LocalDateTime createdTime;
 
+    @Lob
```
Comment (NeuralAlchemist) [inline src/main/java/se/kth/sda/skeleton/comments/Comment.java]: can you remove this? This caused some problems when used in Post for displaying images

**Review by NeuralAlchemist — CHANGES_REQUESTED** [review]
one important change requested but everything looks good otherwise!

File: src/main/java/se/kth/sda/skeleton/comments/Comment.java [hunk 2]
```diff
@@ -46,6 +46,11 @@
...
+    private byte[] fileByte;
+    private String fileType;
+    private String fileName;
```
Comment (eeels22) [inline src/main/java/se/kth/sda/skeleton/comments/Comment.java]: Could we perhaps use the same field names as for Post?
    
```
    private byte[] image;
    private String mediaType;
    private String videoName;
```

File: src/main/java/se/kth/sda/skeleton/comments/CommentService.java [hunk 3]
```diff
@@ -11,14 +11,19 @@
...
 import java.time.LocalDateTime;
+import java.util.Collections;
 import java.util.List;
 
 /**
  * Represents the service layer. It contains defined functionalities according to business logic for Comments.
  */
 @Service
 public class CommentService {
+    private static String videoDirectory = System.getProperty("user.dir") + "/frontend/src/videos/comment";
```
Comment (eeels22) [inline src/main/java/se/kth/sda/skeleton/comments/CommentService.java:26]: Should we add the contents of this folder to .gitignore but keep a README so the folder uploads?

Comment (NeuralAlchemist) [reply src/main/java/se/kth/sda/skeleton/comments/CommentService.java:26]: I think when FE gets implemented we can do that change as it was stuff with react that got bad when fetching those videos

File: src/main/java/se/kth/sda/skeleton/comments/Comment.java (hunk 2 above)
Comment (deciocandido) [reply src/main/java/se/kth/sda/skeleton/comments/Comment.java]: The fields has been rename as Post

**Review by eeels22 — APPROVED** [review]
Looks good, thanks Decio!

**Review by NeuralAlchemist — APPROVED** [review]
Thank you!

**Pull request merged**